### PR TITLE
feat: update the korean translation

### LINKED ISSUE #1700: Help improve Appwrite's Korean language translation ✍️
# Introduction

Did you know Appwrite's Locale API has support for returning responses in your own locale based on the value of the `X-Appwrite-Locale` header? 😮 We already support 56 languages and need your help to improve them! 💪

Your task is to update Appwrite's **Korean** language translations. We recently added new features, so some translations might be missing. You can find a complete in-depth tutorial in our official [tutorial for adding translations](https://github.com/appwrite/appwrite/blob/master/docs/tutorials/add-translations.md).

This is what missing translation would look like:

```json
"emails.magicSession.footer": "",
```

### Tasks summary:

- [ ] Fork & clone Appwrite
- [ ] Update translations for Korean in `app/config/locale/translations/ko.json`
- [ ] Test the translations
- [ ] Push changes and submit pull request

If you need any help, reach out to us on our [Discord server](https://discord.gg/GSeTUeA).

Are you ready to work on this issue? 🤔 Let us know, and we will assign it to you 😊

**Happy Appwriting!**

---

### PR DESCRIPTION
**Pull request by ssong**
## What does this PR do?

Adds new email template translations for Korean and fixes the name for DRC.

Note: had to improvise a bit on `"emails.invitation.subject": "초대장 %s 팀 - %s",` as based on the string parsing order there wasn't really a nice sounding grammatically correct translation. So the translation for the subject currently reads `Invitation %s team - %s`

## Test Plan

Followed @christyjacob4 instructions

```
You can run Appwrite locally and run the e2e tests for the Teams or Accounts service. This will send the emails to the maildev server at localhost:9503
```

## Related PRs and Issues

fixes #1700 

### Have you read the [Contributing Guidelines on issues]

Yes

### CODE REVIEW

File: app/config/locale/translations/ko.json [hunk 1]
```diff
@@ -3,30 +3,30 @@
...
+    "emails.magicSession.thanks": "감사합니다",
+    "emails.magicSession.signature": "{{project}} 팀",
+    "emails.recovery.subject": "비밀번호 재설정",
+    "emails.recovery.hello": "안녕하세요 {{name}}님",
+    "emails.recovery.body": "{{project}}의 비밀번호 재설정을 위해 링크를 클릭하여주세요.",
+    "emails.recovery.footer": "비밀번호 재설정 신청을 하지 않으셨다면 이 메세지를 무시하여주세요.",
+    "emails.recovery.thanks": "감사합니다",
+    "emails.recovery.signature": "{{project}} 팀",
+    "emails.invitation.subject": "초대장 %s 팀 - %s",
+    "emails.invitation.hello": "안녕하세요",
```
Comment (christyjacob4) [inline app/config/locale/translations/ko.json:25]:
```suggestion
    "emails.invitation.hello": "안녕하세요,",
```
Can we make sure punctuation is taken care of in all the strings ?
https://github.com/appwrite/appwrite/blob/master/app/config/locale/translations/en.json#L25

**Review by christyjacob4 — CHANGES_REQUESTED** [review]
Please address the comments

**Comment by christyjacob4** [comment]
I think I made an incorrect reference in my previous comment. Your PR looks good :+1:

**Review by christyjacob4 — APPROVED** [review]

**Comment by eldadfux** [comment]
@ssong THANK YOU! All changes merged 🥳 🇰🇷

Please reach out to me (`eldadfux`) on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! 

As a way of saying thank you, we have just invited you to join the Appwrite organization on GitHub. You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+is%3Aissue+label%3Ahacktoberfest+created%3A%3E2021-01-01+is%3Aopen&type=issues) to help make Appwrite better.

**Comment by eldadfux** [comment]
Thanks for helping @christyjacob4 🙏

**Pull request merged**